lib/exabgp/bgp/message/update/nlri/evpn/mac.py

```diff
@@ -12,6 +12,8 @@
 from exabgp.bgp.message.update.nlri.qualifier.etag import EthernetTag
 from exabgp.bgp.message.update.nlri.qualifier.mac import MAC as MACQUAL
 
+from exabgp.bgp.message.update.nlri.nlri import NLRI
```

Comment:
Hello @tmmorin Thank you for this patch ! Could I just ask you to rebase to the latest code and change this include to be :  "from exabgp.bgp.message.update.nlri import NLRI" otherwise the merge will fail.